activerecord/test/cases/quoting_test.rb

```diff
@@ -189,11 +189,6 @@ def test_quote_string_no_column
         assert_equal "'lo\\\\l'", @quoter.quote('lo\l')
       end
 
-      def test_quote_as_mb_chars_no_column
```

Comment:
I also found another deprecation warning, but I feel the test above tests the same thing

Reply:
No, it tests that `quote` handle `AS::Multibyte::Chars` correctly, which is important.